app/controllers/v0/users_controller.rb

```diff
@@ -16,5 +16,14 @@ def show
     def icn
       render json: { icn: current_user.icn }, status: :ok
     end
+
+    def credential_emails
+      emails = current_user.user_account.user_verifications.each_with_object({}) do |verification, credentials|
+        credentials[verification.user_credential_email.credential_email.to_sym] =
```

Comment:
We'll want the response to look like:
`{ 'idme' => 'email@email.com' }`, right now this renders the response as: `{ 'email1@example.com' => 'email1@example.com' }`